apps/ledger-live-desktop/src/newArch/AnalyticsOptInPrompt/screens/components/RevokeInfoField/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useCallback } from "react";
+import { useTranslation } from "react-i18next";
+import { Flex, Text, Link } from "@ledgerhq/react-ui";
+import { useTheme } from "styled-components";
+import { urls } from "~/config/urls";
+import { openURL } from "~/renderer/linking";
+
+const RevokeInfoField = () => {
+  const { t } = useTranslation();
+  const { colors } = useTheme();
+  const privacyPolicyUrl = urls.privacyPolicy;
```

Comment:
You can use `useLocalizedUrl`, it will allows to have urls localized and based on LLD locale settings